action.yml

```diff
@@ -106,12 +121,25 @@ runs:
             exit 1
         fi
 
+        space_target=$environment_name
```

Comment:
consider renaming space_target to target_space.

Reply:
Done 👍